backend/internal/model/game_test.go

```diff
@@ -138,6 +138,97 @@ func TestGameSubGoal(t *testing.T) {
 	}
 }
 
+func Test_game_UpdateManualGoals(t *testing.T) {
+	game := &game{score: GameScore{0, 0}, manualGoals: ManualGoals{0, 0, 0, 0}}
+
+	type args struct {
+		name                string
+		teamID              int
+		action              string
+		expectedManualGoals ManualGoals
+		expectedError       string
+	}
+	tests := []args{
+		// No error cases
+		{
+			name:                "should increment blue manual goals by 1",
+			teamID:              config.TeamBlue,
+			action:              config.ActionAdd,
+			expectedManualGoals: ManualGoals{1, 0, 0, 0},
```

Comment:
I can see a certain weakness of these tests. You always start with `ManualGoals {0, 0, 0, 0}`. I imagine a function that sets some value to "one" (instead of adding "one"). This function will pass these tests as well.

Reply:
Good point. We have changed it in our tests.